instrumentation/action_view/.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.2
```

Comment:
Mistaken commit?

Reply:
🤦 

Again. I'm going to open a PR to add these to .gitignore 😂

Reply:
Removed in 97e67bf5 and banished forever in #801